### ISSUE TITLE
Export path issue

### ISSUE THREAD

**Issue by lappang-cheung** [mozilla-frontend-infra/react-vnc-display#5]
Getting issues from going 1.0.5 to 1.1.1 where they complained about export file issue.

**Comment by helfi92** [comment]
What are you seeing in the terminal. Also how are you importing the component?

**Comment by lappang-cheung** [comment]
What I am seeing is issue is the following:

Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

And I was creating a simple react-app, so I am importing like
```
import {VncDisplay} from 'react-vnc-display'

class App extends Component {
  render(){
    return(
      <VncDisplay url="testingroute.net:1234/?password=guest" />
    )
  }
}
```

In the older version 1.0.5 was working fine but in order to get it work for me, I had do was

```
import VncDisplay from 'react-vnc-display/build/VncDisplay'
```

**Comment by helfi92** [comment]
Does `import VncDisplay from 'react-vnc-display'` work?

**Comment by lappang-cheung** [comment]
I had the same issue with it

**Comment by helfi92** [comment]
Looking into it.

**Comment by helfi92** [comment]
Ok, the following should work:

```jsx
import VncDisplay from 'react-vnc-display';

// or
const VncDisplay =  require('react-vnc-display').default;
```

I'll update the docs.

**Comment by helfi92** [comment]
I've also published `react-vnc-display@2.0.0` since it was actually a breaking change.

**Comment by lappang-cheung** [comment]
It is working perfectly, I think this issue can be closed
